src/Rings/msolve/msolve.jl

```diff
@@ -0,0 +1,205 @@
+import msolve_jll: libmsolve
+
+export msolve
+
+@doc Markdown.doc"""
+    function get_rational_parametrization(nr::Int32, lens::Array{Int32,1}, cfs::Ptr{BigInt})
+
+Construct the rational parametrization of the solution set computed via msolve.
+
+**Note**: This is an internal function and should only be used inside `msolve()`.
+"""
+function get_rational_parametrization(
+        nr::Int32,
+        lens::Array{Int32,1},
+        cfs::Ptr{BigInt}
+    )
+    C, x  = PolynomialRing(QQ,"x")
+    ctr   = 0
+
+    elim  = 0*x
+    for i in 1:lens[1]
+        elim  +=  BigInt(unsafe_load(cfs, i))*x^(i-1)
+    end
```

Comment:
I think this can be changed to the following more efficient code (but I may have gotten details wrong)
```suggestion
    elim  = C([unsafe_load(cfs, i) for i in 1:lens[1]])
```